build.gradle

```diff
@@ -504,19 +502,6 @@ def tweakStartScript(createScriptTask) {
 
   createScriptTask.unixScript.text = createScriptTask.unixScript.text.replace('BESU_HOME', '\$APP_HOME')
   createScriptTask.windowsScript.text = createScriptTask.windowsScript.text.replace('BESU_HOME', '%~dp0..')
-  // OpenTelemetry Wiring for unix scripts
-  def agentFileName = configurations.javaAgent.singleFile.toPath().getFileName()
-  def unixRegex = $/exec "$$JAVACMD" /$
-  def forwardSlash = "/"
-  def unixReplacement = $/if [ -n "$$TRACING" ];then
- TRACING_AGENT="-javaagent:$$APP_HOME/agent${forwardSlash}${agentFileName}"
-fi
-exec "$$JAVACMD" $$TRACING_AGENT /$
-  createScriptTask.unixScript.text = createScriptTask.unixScript.text.replace(unixRegex, unixReplacement)
-  // OpenTelemetry Wiring for windows scripts
-  def windowsRegex = $/"%JAVA_EXE%" %DEFAULT_JVM_OPTS%/$
-  def windowsReplacement = $/if Defined TRACING (TRACING_AGENT="-javaagent:" "%APP_HOME%\agent\/$ + agentFileName + '")\r\n"%JAVA_EXE%" %DEFAULT_JVM_OPTS% %TRACING_AGENT%'
-  createScriptTask.windowsScript.text = createScriptTask.windowsScript.text.replace(windowsRegex, windowsReplacement)
```

Comment:
What changed so that we no longer need the java agent for OTEL?  I'm not seeing anything code-wise so I'm concerned this is an accidental deletion.

Reply:
The agent inspects the code and injects changes at runtime to create traces from well-known libraries such as SQL querying or HTTP. We now do this manually and in a much more targeted fashion. We no longer need to use automatic instrumentation. Automatic instrumentation could also create extra traces that don't provide as much value.